src/main/java/org/example/api/FileService.java

```diff
@@ -0,0 +1,75 @@
+package org.example.api;
+
+import org.example.model.Directory;
+import org.example.model.MyFile;
+import org.example.util.Data;
+
+public class FileService implements IFileApi {
+    public FileService() {}
+
+    @Override
+    public MyFile addFile(Directory directory, MyFile myFile) {
+        if(!Data.getSystem().contains(directory)) {
+            System.out.println("Directory doesn't exist");
+            return null;
```

Comment:
First of all let's finally get rid of println statements and switch to Exceptions, It would be great to create our custom Exception class that will be thrown in such cases with some errorCode and senseful error message. Why not use simple existing Exception class??? because our custom exception class gives us more flexibility we can add anything we need inside it. In general every application we work on has its own single or multiple exception classes which include many different fields.

Reply:
Yeah sure, implemented exceptions too